src/cmd/commit.ts

```diff
@@ -26,7 +27,12 @@ const commitAndPush = async (values: Record<string, any>, branch: string): Promi
   const d = terminal(`cmd:${cmdName}:commitAndPush`)
   d.info('Committing values')
   const argv = getParsedArgs()
-  const message = isCi ? 'updated values [ci skip]' : argv.message || 'otomi commit'
+  let message: string
```

Comment:
This makes it we dont check double isCi:
```suggestion
  let message = (argv.message as string) || 'otomi commit'
```

Reply:
I was just in the process of refactoring that part. Since we are no longer committing the `legacyRevision` / `defaultRevision`, we need to also add a `--allow-empty`. Therefore this is processed further down and I reverted that line.